Sources/ProjectDescription/OnDemandResourcesTags.swift

```diff
@@ -0,0 +1,12 @@
+/// On-demand resources tags associated with Initial Install and Prefetched Order categories
+public struct OnDemandResourcesTags: Codable, Equatable {
+    /// Initial install tags associated with on demand resources
+    public let initialInstall: [String]?
+    /// Prefetched tag order associated with on demand resources
+    public let prefetchOrder: [String]?
+
+    public init(initialInstall: [String]?, prefetchOrder: [String]?) {
```

Comment:
With the transition from Tuist 3 to Tuist 4 we decided to move towards the usage of static functions as initializers to have more readable code, therefore I'd change this one to a static function named `.tags`

```swift
let target: Target = .target(..., onDemandResourcesTags: .tags(...))
```